src/NUnitFramework/tests/TestContextTests.cs

```diff
@@ -367,7 +367,7 @@ public void FilePathAndDescriptionDoesNotThrow()
 
         [TestCase(null)]
 #if !NETSTANDARD1_3 && !NETSTANDARD1_6
-        [TestCase("bad<>path.png", IncludePlatform = "Windows")]
+        [TestCase("bad<>path.png", IncludePlatform = "Win")]
```

Comment:
Going forward, I would love to see these as an enum of platforms that we support, but for backwards compatibility we could create constants for all the strings for platforms we support and start using those. Obviously not for this pull request, but I would prefer to see `Platform.Windows`.